ui/components/thing-counters.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+export default function ThingCounter({ count, singular, plural }, { router }) {
+  const query = router.location.query;
+  const queryKeys = Object.keys(query);
+  const noun = count === 1 ? singular : plural;
+  const verb = count === 1 ? 'matches' : 'match';
+  const classes = count === 0 ? 'alert p1 m1 border' : '';
+
+  const unfiltered = queryKeys.length === 0 || (queryKeys.length === 1 && queryKeys.indexOf('page') !== -1);
```

Comment:
I decided on:
` const unfiltered = queryKeys.length === 0 || (queryKeys == ['page']);`

The earlier version was overcomplicated, thanks for pointing that out!